Cargo.toml

```diff
@@ -119,7 +119,7 @@ lru_time_cache = "0.10"
 hyper = { version = "0.13", optional = true }
 http = { version = "0.2", optional = true }
 tower = { version = "0.3", optional = true }
-pin-project = "0.4"
+pin-project = "0.4.17"
```

Comment:
Why pinning to v0.4.17?

Reply:
It is because 0.4.17 is the minimal version of the alternative way of project attribute is available. (Also, as it is not "=" requirements, it does not actually pin version of dep)